src/client/utils/parseJwt.js

```diff
@@ -0,0 +1,11 @@
+
+export const parseJwt = (token) => {
+  const base64Url = token.split('.')[1];
```

Comment:
Are we going to use this function?

Reply:
This is not used anywhere now, let's burn it

Reply:
Yeah :guitar: